app/scripts/views/mixins/password-prompt-mixin.js

```diff
@@ -19,8 +19,7 @@ define(function (require, exports, module) {
   const TOOLTIP_MESSAGES = {
     FOCUS_PROMPT_MESSAGE: t('8 characters minimum, but longer if you plan to sync passwords.'),
     INITIAL_PROMPT_MESSAGE: t('A strong, unique password will keep your Firefox data safe from intruders.'),
-    WARNING_PROMPT_MESSAGE: t('This is a common password; please consider another one.'),
-    WARNING_PROMPT_MESSAGE_WITH_LINK: t('This is a common password; please consider another one.')
```

Comment:
the `_WITH_LINK` variant didn't have a link, so I consolidated the two messages.